### PR TITLE
Provide Data Privacy Documentation

### PR DESCRIPTION
**Pull request by MrGeneration**
Hello Ryan,

this pull request provides the all new data privacy feature. 
I already applied the suggestions you provided within Mattermost.

Please have a look at wording, structure and germanized english grammar. :-P
This pull request may not be merged before release of Zammad 3.5.

---

If you have questions, especially about the functionality and want a look and feel, please feel welcome to let me know. I'll gladly explain and show!

Thanks beforehand!

### CODE REVIEW

**Review by rlue — APPROVED** [review]
Just added my QA in 5d1a68b. Have a look and let me know if you have any issues with it.

**Comment by MrGeneration** [comment]
Thanks for your effort, I'll check your QA shortly.
I'll have to have a look onto the gifs you changed, they partly lost color information again.

I'll see if I can improve that.

File: system/data-privacy.rst [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+   a **Delete Organization?** option will be displayed in the user deletion dialog:
+
+   .. figure:: /images/system/data-privacy/delete-organization-option.png
+      :alt: Deleting an organization via the user deletion dialog
+      :align: center
+      :width: 50%
+
+   (If you encounter an error when selecting “Yes” here,
+   make sure there are no pending deletion tasks
+   for other customers from this organization.)
```
Comment (MrGeneration) [inline system/data-privacy.rst:81]: I perosnally think that this text part is not needed.
The reason for this is that Zammad always only provides the organization removal screen if you are removing the last existing user. It does not take in account removal tasks.

I just double checked above.
I'd remove this text? I think you tried to adapt my note that you have to remove all organization members before adding a removal task for the last organization member?

**Comment by MrGeneration** [comment]
Ryan, thank you so much for your time and effort.
I don't have anything to complain - I retouched one of the gifs to improve the quality a bit more. I was failing with that one initially (sorry!).

Apart from that, please have a look at my one comment regarding organization removal, after that we should be good to go. 💪

File: system/data-privacy.rst (hunk 1 above)
Comment (rlue) [reply system/data-privacy.rst:81]: Ah, I see what you mean now! How's this?

```suggestion
   (If this option does not appear,
   make sure there are no pending deletion tasks
   for other customers from this organization.)
```

**Comment by MrGeneration** [comment]
Thanks!

**Pull request merged**